### ISSUE TITLE
Hello， I wonder to know why this training accuracy is about 0.5, but the accuracy in the C3D paper  is about 0.8?   thanks!

### ISSUE THREAD

**Issue by ARQlalala** [chuckcho/video-caffe#101]
Please use the [caffe-users list](https://groups.google.com/forum/#!forum/caffe-users) for usage, installation, or modeling questions, or other requests for help.
_Do not post such requests to Issues._ Doing so interferes with the development of Caffe.

Please read the [guidelines for contributing](https://github.com/BVLC/caffe/blob/master/CONTRIBUTING.md) before submitting this issue.

### Steps to reproduce

If you are having difficulty building Caffe or training a model, please ask the caffe-users mailing list. If you are reporting a build error that seems to be due to a bug in Caffe, please attach your build configuration (either Makefile.config or CMakeCache.txt) and the output of the make (or cmake) command.

### Your system configuration
Operating system:
Compiler:
CUDA version (if applicable):
CUDNN version (if applicable):
BLAS:
Python or MATLAB version (for pycaffe and matcaffe respectively):

**Comment by chuckcho** [comment]
@ARQlalala 50% accuracy you're getting is when you train from a scratch, while 80% accuracy in the paper is based on fine-tuning from a pre-trained model (Sport1m).

**Comment by chuckcho** [comment]
See figure 2 in the [original paper](http://vlg.cs.dartmouth.edu/c3d/c3d_video.pdf) which is comparable to ~50% number you're getting.
